### PR TITLE
Chore(API):Adding API tests for dashbaord

### PR DESCRIPTION
**Pull request by MedWassCh**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

## Summary by CodeRabbit

- **Tests**
	- Introduced an environment configuration for dashboard API testing to enhance connection parameters, endpoints, and credentials management.

### CODE REVIEW

**Review by jdelpierre — DISMISSED** [review]

**Comment by jdelpierre** [comment]
collection folder : dashboard, the "o" is not at the good place

File: centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,1618 @@
...
...  \"options\":{\r\n                \"resourceType\":\"host\",\r\n                \"statuses\":[\r\n                   \"up\",\r\n                   \"down\"\r\n                ],\r\n                \"states\":[\r\n                ],\r\n                \"sortBy\":\"status_severity_code\",\r\n                \"tiles\":100,\r\n                \"refreshInterval\":\"default\",\r\n                \"description\":{\r\n                   \"content\":\"{\\\"root\\\":{\\\"children\\\":[{\\\"children\\\":[{\\\"detail\\\":0,\\\"format\\\":0,\\\"mode\\\":\\\"normal\\\",\\\"style\\\":\\\"\\\",\\\"text\\\":\\\"Descirption for status grid\\\",\\\"type\\\":\\\"text\\\",\\\"version\\\":1}],\\\"direction\\\":\\\"ltr\\\",\\\"format\\\":\\\"\\\",\\\"indent\\\":0,\\\"type\\\":\\\"paragraph\\\",\\\"version\\\":1}],\\\"direction\\\":\\\"ltr\\\",\\\"format\\\":\\\"\\\",\\\"indent\\\":0,\\\"type\\\":\\\"root\\\",\\\"version\\\":1}}\",\r\n                   \"enabled\":true\r\n                },\r\n                \"name\":\"Title For status grid\",\r\n                \"openLinksInNewTab\":true\r\n             }\r\n          },\r\n          \"widget_type\":\"/widgets/statusgrid\"\r\n       }\r\n    ]\r\n }",
```
Comment (mdebauchez) [inline centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json:815]: Description mispelled in "Descirption for status"

File: centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,1618 @@
...
+			"listen": "test",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		}
+	],
+	"variable": [
```
Comment (mdebauchez) [inline centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json:1580]: You can remove variable names in the collection to reduce the number of unused rows

File: centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,1618 @@
...
+					"response": []
+				},
+				{
+					"name": "Getting dashboard with status grid widget",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"pm.collectionVariables.set(\"dashboardName\", \"Dashboard-Test-API-1002\");\r",
```
Comment (mdebauchez) [inline centreon/tests/rest_api/collections/dashbaord/Dashboards.postman_collection.json:843]: There is no need to redefine variables if they are saved as collectionVariables
Same comment for the following pre-requests

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**